static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -5,383 +5,331 @@ import {
     Component,
     styled,
 } from '@guardian/dotcom-rendering/packages/guui';
-import palette, {
-    pillarsHighlight,
-} from '@guardian/dotcom-rendering/packages/pasteup/palette';
+import palette from '@guardian/dotcom-rendering/packages/pasteup/palette';
 import {
-    mobileMedium,
-    mobileLandscape,
-    phablet,
-    tablet,
     leftCol,
     wide,
-    mobile,
-    desktop,
 } from '@guardian/dotcom-rendering/packages/pasteup/breakpoints';
 
-import download from 'svgs/journalism/audio-player/download.svg';
 import pauseBtn from 'svgs/journalism/audio-player/pause-btn.svg';
 import playBtn from 'svgs/journalism/audio-player/play-btn.svg';
-import volume from 'svgs/journalism/audio-player/volume.svg';
+import volumeOn from 'svgs/journalism/audio-player/volume-on.svg'; // eslint-disable-line
+import volumeOff from 'svgs/journalism/audio-player/volume-on.svg'; // eslint-disable-line
 import fastBackward from 'svgs/journalism/audio-player/fast-backward.svg';
 import fastForward from 'svgs/journalism/audio-player/fast-forward.svg';
 import fastBackwardActive from 'svgs/journalism/audio-player/fast-backward-active.svg';
 import fastForwardActive from 'svgs/journalism/audio-player/fast-forward-active.svg';
 
 import waveW from 'svgs/journalism/audio-player/wave-wide.svg';
-import { formatTime, sendToOphan, checkForTimeEvents } from './utils';
+import { sendToOphan, checkForTimeEvents } from './utils';
```

Comment:
Meant to mention this before. But the `checkForTimeEvents` function is too restrictive and should be replaced with `monitorPercentPlayed`. Should also move into utils for consistency https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/audio/inArticleDataEvents.js